docs/src/data/frameworks.ts

```diff
@@ -13,18 +19,21 @@ const BASE_FRAMEWORK_DISPLAY_NAMES = {
   vue: 'Vue',
 };
 
-export const FRAMEWORK_DISPLAY_NAMES = isReactNativeEnabled
+export const FRAMEWORK_DISPLAY_NAMES = IS_REACT_NATIVE_ENABLED
   ? {
       ...BASE_FRAMEWORK_DISPLAY_NAMES,
       'react-native': 'React Native',
     }
   : BASE_FRAMEWORK_DISPLAY_NAMES;
 
+// React Native requires direct installation of dependencies with native modules
+export const REACT_NATIVE_DEPENDENCIES =
+  '@react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill react-native-safe-area-context';
```

Comment:
@calebpollman  @dbanksdesign What do you folks think of having this be multiple lines with `\`?
```suggestion
  '\
  @react-native-community/netinfo \
  @react-native-async-storage/async-storage \
  react-native-get-random-values \
  react-native-url-polyfill react-native-safe-area-context';
```

It gets pretty wide on widescreen monitors:
<img width="2495" alt="image" src="https://user-images.githubusercontent.com/6165315/195215277-8ea2b0db-05ee-4198-a3a4-b3a9b7701e7e.png">